templates/items/ffg-specialization-sheet.html

```diff
@@ -36,7 +36,8 @@
             <div class="talent-hidden">
               <input class="talent-hidden" type="text" name="data.talents.{{key}}.name" value="{{talent.name}}" />
               <input class="talent-hidden" type="text" name="data.talents.{{key}}.description" value="{{talent.description}}" />
-              <!-- <input class="talent-hidden" type="text" name="data.talents.{{key}}.activation" value="{{talent.activation}}" /> -->
+              <input class="talent-hidden" type="text" name="data.talents.{{key}}.activation" value="{{talent.activation}}" />
+              <input class="talent-hidden" type="text" name="data.talents.{{key}}.activationLabel" value="{{talent.activationLabel}}" />

```

Comment:
Be wary of duplicate inputs for the same objects, it causes crazy behaviour with infinitely increasing commas.

Reply:
Yep, I ran into that early on, when it reduces the formobjects in update, duplicate inputs turn into an array of values which is added.

Reply:
Ahh I guess I misses the activationlabel input above.